utils/cliutils/commandsflags.go

```diff
@@ -1418,6 +1432,26 @@ var flagsMap = map[string]cli.Flag{
 		Name:  Status,
 		Usage: "[Default: false] Set to true to show the status of the transfer-files command currently in progress.` `",
 	},
+	branch: cli.StringFlag{
+		Name:  branch,
+		Usage: "[Optional] Branch name to filter.` `",
+	},
+	pipelineName: cli.StringFlag{
+		Name:  pipelineName,
+		Usage: "[Optional] Pipeline name to filter.` `",
+	},
+	monitor: cli.BoolFlag{
+		Name:  monitor,
+		Usage: "[Default: false] Monitor pipeline status.` `",
+	},
+	repository: cli.StringFlag{
+		Name:  repository,
+		Usage: "[Optional] Repository name to filter resource.` `",
+	},
+	multiBranch: cli.StringFlag{
+		Name:  multiBranch,
+		Usage: "[Default: true] Multi branch to filter multi branches from single branch.` `",
```

Comment:
We're trying to avoid boolean flags with default "true" values whenever possible.
Can we change the flag to `singleBranch` and default false?